software/directory-lister.yml

```
@@ -1,10 +1,10 @@
-name: DirectoryLister
+name: Directory Lister
 website_url: https://www.directorylister.com/
 description: Simple PHP based directory lister that lists a directory and all its sub-directories and allows you to navigate there within.
 licenses:
   - MIT
 platforms:
-  - PHP
+  - PHP/Docker
```

This will fix the build but I cant seem to find docker listed in https://github.com/DirectoryLister/DirectoryLister#requirements
```suggestion
  - PHP
  - Docker
```

---

I will update this.

Docker isn't a _requirement_ for running the application itself (which is what that section represents). However, there's a Docker container available ([docs](https://docs.directorylister.com/docker)).